src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```diff
@@ -173,14 +175,52 @@ admin_orders_update_invoice_note:
     orderId: \d+
     orderInvoiceId: \d+
 
-admin_search_product_for_order_creation:
-  path: /search
-  methods: [GET]
-  defaults:
-    _controller: PrestaShopBundle:Admin/Sell/Order/Order:searchProducts
-
 admin_orders_generate_invoice:
   path: /{orderId}/invoice
   methods: [POST]
   defaults:
     _controller: PrestaShopBundle:Admin/Sell/Order/Order:generateInvoice
+
+admin_orders_add_product:
+  path: /{orderId}/products
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Order:addProduct
+    _legacy_controller: AdminOrders
+  requirements:
+    orderId: \d+
+  options:
+    expose: true
+
+admin_orders_delete_product:
+  path: /{orderId}/products/{orderDetailId}/delete
+  methods: [POST]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Order:deleteProduct
+    _legacy_controller: AdminOrders
+  requirements:
+    orderId: \d+
+    orderDetailId: \d+
+  options:
+    expose: true
+
+admin_orders_get_prices:
+  path: /{orderId}/prices
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Order:getPrices
+    _legacy_controller: AdminOrders
+  requirements:
+    orderId: \d+
+  options:
+    expose: true
+
+admin_orders_get_paginated_products:
+  path: /{orderId}/products
+  methods: [GET]
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Order/Order:getPaginatedProductsAction
```

Comment:
Missing `_legacy_controller` for security purposes

Reply:
:ballot_box_with_check: